### ISSUE TITLE
A bot exists to close stale GitHub issues

### ISSUE THREAD

**Issue by izgeri** [cyberark/conjur#787]
•	Create a bot that closes stale issues
- First, no activity for a certain time adds a `stale` label which pushes a notification to the issue opener
- If stale with no further activity for a period of time, the issue is closed; if activity occurs `stale` is auto-removed

Depends on #781

**Comment by izgeri** [comment]
this is a duplicate of cyberark/community#84 - closing as work will be tracked in the community repo from now on.
